float64 first_item
float64 last_item